packages/server/test/unit/browsers/browsers_spec.ts

```diff
@@ -417,43 +468,108 @@ describe('lib/browsers/index', () => {
       // Stub to speed up test, we don't care about the delay
       sinon.stub(Promise, 'delay').resolves()
 
-      return browsers.open({ name: 'electron', family: 'chromium' }, { url }, null, ctx).then(browsers.close).then(() => {
+      return browsers.open({ name: 'electron', family: 'chromium' } as any, { url } as any, null, ctx).then(browsers.close).then(() => {
         expect(ctx.coreData.didBrowserPreviouslyHaveUnexpectedExit).eq(true)
       })
     })
   })
-})
 
-// Ooo, browser clean up tests are disabled?!!
-
-// it "calls onBrowserClose callback on close", ->
-//   onBrowserClose = sinon.stub()
-//   browsers.launch("electron", @url, {onBrowserClose}).then ->
-//     Windows.create.lastCall.args[0].onClose()
-//     expect(onBrowserClose).to.be.called
-//
-// it "calls onBrowserOpen callback", ->
-//    onBrowserOpen = sinon.stub()
-//    browsers.launch("electron", @url, {onBrowserOpen}).then =>
-//      expect(onBrowserOpen).to.be.called
-//
-// it "waits a second to give browser time to open", ->
-//   browsers.launch("electron").then ->
-//     expect(Promise.delay).to.be.calledWith(1000)
-//
-// it "returns 'instance'", ->
-//   browsers.launch("electron").then (instance) ->
-//     expect(instance.kill).to.be.a("function")
-//     expect(instance.removeAllListeners).to.be.a("function")
-//
-// it "closes window on kill if it's not destroyed", ->
-//   @win.isDestroyed.returns(false)
-//   browsers.launch("electron").then (instance) =>
-//     instance.kill()
-//     expect(@win.close).to.be.called
-//
-// it "does not close window on kill if it's destroyed", ->
-//   @win.isDestroyed.returns(true)
-//   browsers.launch("electron").then (instance) =>
-//     instance.kill()
-//     expect(@win.close).not.to.be.called
+  context('browser cleanup', () => {
+    it('calls onBrowserClose callback on close', async () => {
```

Comment:
these shouldn't need to be async in their current state unless you want to rework the function to await `browsers.open`
```suggestion
    it('calls onBrowserClose callback on close', () => {
```